### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][NVDA][MAS4.2.3][IntelliJ][Wipro]:Table commands are not working for tables present in create service principle window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1489]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Open your project with IntelliJ IDEA
3. Tab to Tools menu, point to "Azure", and then select "Azure Sign In".
4. In the Azure Sign In window, select "Automated" radio button, and navigate to "New" button and select it.
5. In the Azure Login Dialog window, enter your "Azure credentials", and then select "Sign in" button.
6. After sign In Create Authentication Files window will opened .
7. In "Create Authentication files window" navigate and select the "subscriptions" that you want to use, choose your "destination directory", and then select Start.
8. In the Create Service Principle status, press tab to navigate tables then try to access the table cells by using table commands(CTRL+Alt+Arrow keys) then observe the NVDA Behavior

**Expected Result:**
**NVDA Behavior:**
By using below table command keys user should be able to access the table and NVDA should announced the Value present in the table cell then row position header name and column position
 Ctrl + Alt + Right Arrow-> Cell to right
Ctrl + Alt +Left Arrow->Cell to left
Ctrl + Alt +Down Arrow-> Cell below
Ctrl + Alt +Up Arrow->Cell above

**Actual Result:**
**NVDA Behavior:**
NVDA is not announcing anything by using table command keys and table keys are not working
**Jaws Behavior:**  Jaws is announcing as expected
Not Selected Creating Azure Active Directory application
Result
Not Selected SUCCESSFUL
Column 2
Details
Not Selected AppID: 6c3860f0-c421-4fd6-9783-034c43b9d522; appObjectId: 11be16d6-73c7-4eef-b829-ad4a4f799b47
Column 3

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![create service principle status-table](https://user-images.githubusercontent.com/22956130/36135351-97ac36da-10b0-11e8-83ba-4b4d18aae18f.jpg)
![table command keys are not working in create service principle](https://user-images.githubusercontent.com/22956130/36135355-9fbb528e-10b0-11e8-882f-76d23f84823b.jpg)

**Comment by Eskibear** [comment]
This is a standard IntelliJ IDEA table, out of the toolkit's control.

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749955
